lib/main.js

```diff
@@ -70,21 +66,13 @@ define(['moment', 'utils/router', 'leaflet', 'gui', 'helper', 'utils/language'],
       var language = new Language(config);
       var router = new Router(language);
 
-      var urls = [];
-
-      if (typeof config.dataPath === 'string' || config.dataPath instanceof String) {
-        config.dataPath = [config.dataPath];
-      }
-
-      for (var i in config.dataPath) {
-        if (config.dataPath.hasOwnProperty(i)) {
-          urls.push(config.dataPath[i] + 'meshviewer.json');
-        }
-      }
+      config.dataPath.forEach(function (d, i) {
```

Comment:
need change of `config.json` now it has always to be an array of urls

Reply:
yeah, nice catch, didn't submit the testurl.